core.md

```diff
@@ -0,0 +1,111 @@
+# Governance of the NeoForged project
+
+The NeoForged project is split between [github](https://github.com/NeoForged) and [discord](https://discord.gg/UvedJ9m). It is a community of volunteers, maintaining a free and open source ecosystem of modifications to the game Minecraft.
+
+### What this document is
+
+This document is:
+ * A non-exhaustive list of the working principles of the project.
+ * Intended to help direct the project in times of uncertainty.
+
+This document is built on the fundamental principle of "people generally want to help, so we should not get in the way of it".
+
+The core ethic is "people over processes", so this document is intentionally light on processes - situations that come up are rarely one-size-fits-all, and should be treated on a case by case basis, with the best process to fit the person.
+
+### What this document is not
+
+This document is not:
+ * Intended to enforce rules on all members.
+ * Intended to catch all possibilities.
+ * Intended to prevent "rules lawyering".
+
+The code of conduct of the project is taken to be the content of [the Contributor Covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/), so this document is not the code of conduct.
```

Comment:
```suggestion
The Code of Conduct of the project is taken to be the content of [the Contributor Covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/), so this document is not the Code of Conduct.
```
We should still probably have our own CoC

Reply:
If there's a pressing need over the Contributor Covenant, I can see that, yeah.

I'm not sure if there's anything we need that isn't covered, to be honest. Nobody is above the rules.